File: api-examples/cat/search/python/attribute_search_spatial_circle_responseFilter.py
import requests

url = "https://<catalogue-server-ip>/catalogue/v1/search?attribute-name=(tags)&attribute-value=((pollution))&lat=12.273737&lon=78.37475&radius=200000&attribute-filter=(tags,id)"

payload = {}
headers= {}

response = requests.request("GET", url, headers=headers, data = payload)

print(response.text.encode('utf8'))

File: api-examples/cat/search/java/attribute_search_spatial_circle_responseFilter.java
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
Request request = new Request.Builder()
  .url("https://<catalogue-server-ip>/catalogue/v1/search?attribute-name=(tags)&attribute-value=((pollution))&lat=12.273737&lon=78.37475&radius=200000&attribute-filter=(tags,id)")
  .method("GET", null)
  .build();
Response response = client.newCall(request).execute();
